### PR TITLE
https://issues.redhat.com/browse/ACM-9294 Add import step

### PR DESCRIPTION
**Pull request by oafischer**
2.9 only

### CODE REVIEW

File: clusters/cluster_lifecycle/import_ocp.adoc [hunk 1]
```diff
@@ -159,3 +159,5 @@ spec:
 ----
 oc get infraenv -n managed-cluster some-other-infraenv -ojson | jq ".status.<url>" --raw-output | xargs curl -k -o /storage0/isos/some-other.iso
 ----
+
+. *Optional:* If you want to use {product-title-short} features, such as policies, on your {ocp-short} cluster, create a `ManagedCluster` resource. Make sure that the name of your `ManagedCluster` resource matches the name of your `ClusterDeplpoyment` resource.
```
Comment (jelkosz) [inline clusters/cluster_lifecycle/import_ocp.adoc]: I would also mention that if you dont have a ManagedCluster resource, the cluster will show up as "detached" in the UI.

Comment (oafischer) [reply clusters/cluster_lifecycle/import_ocp.adoc]: Updated!

**Comment by jelkosz** [comment]
thanks, lgtm

**Review by dockerymick — APPROVED** [review]
Thanks Oliver!

**Pull request merged**